src/sage/graphs/generators/families.py

```diff
@@ -513,12 +513,12 @@ def BalancedTree(r, h):
     OUTPUT:
 
     The perfectly balanced tree of height `h \geq 1` and whose root has
-    degree `r \geq 2`. A ``NetworkXError`` is returned if `r < 2` or
+    degree `r \geq 2`. A :class:`~networkx.NetworkXError`` is raised if `r < 2` or
     `h < 1`.
 
     ALGORITHM:
 
-    Uses `NetworkX <http://networkx.lanl.gov>`_.
+    Uses the :ref:`NetworkX <spkg_networkx>` function :func:`~networkx.balanced_tree`.
```

Comment:
This adds a link to the sagemath page on the Networkx package but not to the function `balanced_tree`

https://deploy-preview-37575--sagemath.netlify.app/html/en/reference/graphs/sage/graphs/generators/families

Reply:
Fixed in 93b99874d47 (I hope)